eventstore/models.py

```diff
@@ -107,6 +107,14 @@ class MSISDNSwitch(models.Model):
     data = JSONField(default=dict, blank=True, null=True)
 
 
+class DeliveryFailures(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    contact_id = models.CharField(max_length=255, blank=True)
```

Comment:
I think we can make the `contact_id` the primary key here, and remove the `id` field, because each contact should only have 1 of these to keep track of the delivery failures for that contact, so it is a unique identifier.

Reply:
I think we need a `primary_key=True` here, to ensure that the contact_id is a primary key (otherwise it is possible to have duplicates)